semgrep-core/src/tainting/Dataflow_tainting.ml

```diff
@@ -212,6 +212,11 @@ let merge_source_sink_mvars env source_mvars sink_mvars =
     (* The union of both sets, but taking the sink mvars in case of collision. *)
     sink_biased_union_mvars source_mvars sink_mvars
 
+let partition_mutable_sources sources_matches =
```

Comment:
Personally I think `mutating_sources` is better? Mutable to me sounds like we're trying to identify if the source is mutable (e.g. a list in Python)

Reply:
Agree, I will rename it.